src/Platform/Microsoft.Testing.Platform/Services/ITestApplicationModuleInfo.cs

```diff
@@ -11,8 +11,6 @@ internal interface ITestApplicationModuleInfo
 
     bool IsAppHostOrSingleFileOrNativeAot { get; }
 
-    string GetCommandLineArguments();
```

Comment:
Is the implementation in `CurrentTestApplicationModuleInfo` unused as well? In this case could we clean it up there as well? Thanks!